biggraphite/accessor.py

```diff
@@ -315,26 +320,8 @@ def insert_downsampled_points_async(self, metric, datapoints, on_done=None):
         self._check_connected()
 
     @abc.abstractmethod
-    def map(self, callback, start_key=None, end_key=None, shard=0, nshards=1, errback=None):
-        """Call callback on each metric.
-
-        This operation can potentially be very slow.
-
-        This will list each metrics matching the arguments and call
-        `callback(metric, done, total)`.
-
-        Args:
-          callback: callable(metric: Metric, done: int, total: int)
-          errback: callable(metric: string) when a corrupted metric is found
-          start_key: string, start at key >= start_key.
-          end_key: string, stop at key < end_key.
-          shard: int, shard to repair.
-          nshards: int, number of shards.
-
-        """
-        assert shard >= 0
-        assert nshards > 0
-        assert shard < nshards
+    def map(self, namespaces, **kwargs):
```

Comment:
keep the idiomatic map function here. If it's not generic, we may move it to cassandra specific.

Reply:
Done